### PR TITLE
feat(portals-admin): Fix clients loading state and button loading state

### PR DESCRIPTION
**Pull request by snaerseljan**
# Fixes for ids admin

## What

Fix create client loading state. I added a special hook to handle loading state in RR form submissions.
Fixed refetching clients when navigating back.
Fix unused ref which was causing warning in UI.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by saevarma — APPROVED** [review]
💯

File: libs/island-ui/core/src/lib/AsyncSearch/shared/Label/Label.tsx [hunk 1]
```diff
@@ -9,10 +9,11 @@ export interface LableProps extends LabelHTMLAttributes<HTMLLabelElement> {
 }
 
 export const Label = forwardRef<HTMLLabelElement, LableProps>(
-  ({ hasError, color, children, ...props }) => {
```
Comment (oddsson) [inline libs/island-ui/core/src/lib/AsyncSearch/shared/Label/Label.tsx:12]: Is this `ref` never used or am I missing something 🤔

**Review by oddsson — COMMENTED** [review]
One question

File: libs/island-ui/core/src/lib/AsyncSearch/shared/Label/Label.tsx (hunk 1 above)
Comment (snaerseljan) [reply libs/island-ui/core/src/lib/AsyncSearch/shared/Label/Label.tsx:12]: I image this is never used but the forwardRef was there initially without a ref. I could also just remove the HOC but this felt a little safer.

Comment (snaerseljan) [reply libs/island-ui/core/src/lib/AsyncSearch/shared/Label/Label.tsx:12]: @oddsson can you re review or approve if you are satisfied :)

**Review by oddsson — APPROVED** [review]

**Pull request merged**